lib/workers/api_body_title.js

```diff
@@ -0,0 +1,165 @@
+var semver = require('semver');
```

Comment:
This whole file is mostly a copy of "api_param_title.js". You should base on the existing code (maybe including api_param_title.js) and apply only small changes. 

Having the same code 3 is very bad